pages/src/common/alert.less

```diff
@@ -18,14 +20,14 @@
   }
 
   &-warning {
-    color: #856404;
-    background-color: #fff3cd;
-    border-color: #ffeeba;
+    color: @primary-blue;
+    background-color: rgba(0, 151, 231, 0.2);
+    border-color: rgba(0, 151, 231, 0.25)
```

Comment:
Look like we need mixin to change color opacity.

Reply:
Less have a built-in function to do that, just in case: `fade(@original-color, 50% /*opacity*/)`